app/reducers/currentpatient.js

```diff
@@ -2,20 +2,20 @@ import axios from 'axios';
 
```

Comment:
I had to change the action creator adding payload. Also in the combiner I had to change currentpatient to currentPatient since all its referred to as currentPatient on our props